galata/src/helpers/sidebar.ts

```diff
@@ -267,6 +267,45 @@ export class SidebarHelper {
     });
   }
 
+  /**
+   * Set the sidebar width
+   *
+   * @param width Sidebar width in pixels
+   * @param side Which sidebar to set: 'left' or 'right'
+   */
+  async setWidth(
```

Comment:
Fine to add it in 4.2.0, but let's remove it on backport to 4.1.x to avoid expanding API surface in patch releases.